package.json

```diff
@@ -2,37 +2,38 @@
   "scripts": {
     "test": "BABEL_ENV=test karma start --single-run",
     "build": "webpack",
-    "lint": "eslint --ext mjs,jsx,js scanomatic/ui_server_data/js/ccc scanomatic/ui_server_data/js/specs"
+    "lint": "eslint --ext mjs,jsx,js scanomatic/ui_server_data/js/ccc scanomatic/ui_server_data/js/tests"
   },
   "devDependencies": {
     "audit": "^0.0.6",
     "babel-core": "^6.26.0",
-    "babel-eslint": "^8.0.2",
+    "babel-eslint": "^10.1.0",
     "babel-loader": "^7.1.2",
-    "babel-plugin-istanbul": "^4.1.5",
+    "babel-plugin-istanbul": "^4.1.6",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "css-loader": "^0.28.7",
+    "css-loader": "^5.0.0",
     "enzyme": "^3.1.0",
     "enzyme-adapter-react-16": "^1.0.2",
-    "eslint": "^4.11.0",
+    "eslint": "^4.19.1",
```

Comment:
This is crucial. It could be further updated, but `5.x.x` requires an update of `eslint-config-airbnb` which in turn causes a lot of lint issues, that would make this PR very messy. I intend to try this, but would prefer to do it in a separate PR.